generator/web/src/main/kotlin/me/kcra/takenaka/generator/web/pages/classPage.kt

```diff
@@ -582,3 +625,43 @@ fun Type.format(remapper: ContextualElementRemapper, isVarargs: Boolean = false)
         else -> className
     }
 }
+
+/**
+ * Formats a modifier integer into a string.
+ *
+ * @param mask the modifier mask (you can get that from the [Modifier] class)
+ * @param classType the type of the class or the type of the class containing the member
+ * @return the modifier string, **may end with a space**
+ */
+fun Int.formatModifiers(mask: Int = 0, classType: ClassType): String = buildString {
```

Comment:
this should only concern a member parent in my opinion, it's not necessary when formatting modifiers for a class itself

```suggestion
fun Int.formatModifiers(mask: Int = 0, parentType: ClassType = ClassType.CLASS): String = buildString {
```